app/lib/katello/resources/registry.rb

```diff
@@ -18,13 +18,13 @@ def self.get(path, headers = {:accept => :json})
       end
 
       class RegistryResource < HttpResource
-        if SETTINGS[:katello][:registry]
-          cfg = SETTINGS[:katello][:registry]
-          url = cfg[:url]
+        if SETTINGS[:katello][:container_image_registry]
+          cfg = SETTINGS[:katello][:container_image_registry]
+          url = cfg[:crane_url]
```

Comment:
This implies it only works with crane and no other container registry. Is that correct?

Reply:
If it only works with crane, you might as well name the name the section crane, right? Registry is singular so a user wouldn't expect to be able to use more than one anyway.

Reply:
@ekohl No, let's keep it as crane since that makes it very clear what service is being contacted. There is a chance there will be an alternative registry implementation in the coming months as well so this will help distinguish it. Thanks!

Reply:
I thought this had all been finalized on the related PR too. https://github.com/theforeman/puppet-katello/pull/243#discussion_r199203794

Reply:
I'd like to know where it's going and get it right the first time. I would expect registries to behave in the same way so they can be used interchangeable but perhaps this is not the reality.

So currently the format is:

```yaml
:container_image_registry:
  :crane_url: ""
  :ca_ca_cert_file: ""
```

Now if another registry is added, how will this be configured?

```yaml
:container_image_registry:
  :registry_type: "crane" # or my_registry
  :crane_url: ""
  :ca_ca_cert_file: ""
  :my_registry_type_url: ""
  :my_registry_type_ca_cert_file: ""
```